content/en/docs/tasks/manage-kubernetes-objects/kustomization.md

```diff
@@ -113,10 +113,10 @@ The generated ConfigMap is:
 ```yaml
 apiVersion: v1
 data:
-    FOO=Bar
+  FOO: Bar
```

Comment:
I _think_ is right if using Kustomize with its dotenv parsing mode. I haven't checked.